### ISSUE TITLE
Air Gap Install pipeline for registry:2 images error

### ISSUE THREAD

**Issue by ntfs32** [rancher/rancher#19716]
What kind of request is this (question/bug/enhancement/feature request):
Bug

Steps to reproduce (least amount of steps as possible):
download save images script
```
# wget https://github.com/rancher/rancher/releases/download/v2.2.2/rancher-images.txt
# wget https://github.com/rancher/rancher/releases/download/v2.2.2/rancher-load-images.sh
# https://github.com/rancher/rancher/releases/download/v2.2.2/rancher-save-images.sh
# chmod +x *.sh
```

after run rancher-save-images.sh  we can't load images to private registry
private registry hasn't any default registry named: reg.xxx.com/registry:2

in file: https://github.com/rancher/rancher/blob/v2.2.2/pkg/image/resolve.go
all images like registry:2,rancher will load by name: `rancher/registry:2`

so i think we shoud update:
 `rancher-load-images.sh` tag `registry:2` to `private.xxx.com/rancher/registry:2` 
and push it.

**Comment by jiaqiluo** [comment]
The bug fix is not on v2.3.0-alpha2

**Comment by deniseschannon** [comment]
@jiaqiluo it looks like this was merged awhile ago. can you re-test using v2.3.0-alpha5?

**Comment by jiaqiluo** [comment]
The bug fix is validated in Rancher: v2.3.0-alpha5 

Result:
- if the image name is `abc:2`, it will be tagged and pushed to the private registry as `private-registry/abc:2`
